templates/explore/user_list.tmpl

```diff
@@ -7,11 +7,11 @@
 			<div class="flex-item-main">
 				<div class="flex-item-title">
 					{{if .IsOrganization}}
-						<span data-tooltip-content title="{{ctx.Locale.Tr "explore.user_is_organization" .DisplayName}}">{{svg "octicon-organization"}}</span>
+						<span class="tw-flex tw-justify-center tw-flex-wrap" data-tooltip-content title="{{ctx.Locale.Tr "explore.user_is_organization" .DisplayName}}">{{svg "octicon-organization"}}</span>
```

Comment:
Not `tw-items-center` instead of `tw-justify-center`? The latter would center horizontally.

Reply:
I have no idea.
I simply translated your suggestion into tailwind.
Also, I see no improvement when adding `tw-items-center`…

Reply:
Ah, sorry `gt-ac` is actually `tw-content-center`.

Reply:
Still no difference

Reply:
I will check later. Maybe we should also move the icon into second row and add a User/Organization/Bot text besides it.

Reply:
(the text is currently a tooltip)